drivers/i2c/Kconfig.pca9542a

```diff
@@ -0,0 +1,27 @@
+# pca9542a
```

Comment:
If possible uppercase with a 1-line description. I.e. the same as you added below. Not very important though!

```
# NXP PCA9542A 2-channel multiplexer
```